src/components/EventList.spec.tsx

```diff
@@ -46,33 +63,67 @@ describe('EventList', () => {
     });
 
     it('contains a sign-up button for each event', () => {
+        const spyOnSubmit = cy.spy();
+
         mountWithProviders(
-            <EventList events={ dummyEvents } org={ dummyOrg }/>,
+            <EventList
+                eventResponses={ dummyEventResponses }
+                events={ dummyEvents }
+                onEventResponse={ spyOnSubmit }
+                org={ dummyOrg }
+            />,
         );
 
-        cy.contains('misc.eventList.signup');
+        //Checks for buttons on all events
+        cy.findByText('misc.eventList.signup');
+
+        //Tests button on a single event
+        cy.findByText('misc.eventList.signup')
+            .eq(0)
+            .click()
+            .then(() => {
+                expect(spyOnSubmit).to.be.calledOnce;
+            });
     });
 
     it('contains a button for more info on each event', () => {
+        const spyOnSubmit = cy.spy();
```

Comment:
Why the spy here?

Reply:
Seems like I missed this when removing spies, will remove.